lib/MusicBrainz/Server/Controller/Role/EditRelationships.pm

```diff
@@ -461,6 +461,20 @@ role {
                     $target = $field->{forward} ? $relationship->entity1 : $relationship->entity0;
                 }
 
+                if ($relationship) {
+                    my $relationship_source = $field->{forward}
+                        ? $relationship->entity0
+                        : $relationship->entity1;
```

Comment:
Just to make sure, I understand this should cover cases where the direction was switched, so the old source is now the target?

Reply:
It should (that was the point of the check, at least), but I'll make sure we have a test for this if we don't.

Reply:
Turns out we didn't, and there was a bug here. 🫣 Thanks for making me check.